### PR TITLE
Constants fix

### PR DESCRIPTION
**Pull request by ehsteve**
This is to deal with #1748. Not a perfect solution but it is a step in the right direction.

### CODE REVIEW

File: astropy_helpers [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 3a1f7998e00d29db6a9e66465f21fd2d635f44cd
+Subproject commit d8f48901442e4056879c4249e73ecd7d04a28282
```
Comment (Cadair) [inline astropy_helpers]: this should not be here.

File: sunpy/sun/constants.py [hunk 2]
```diff
@@ -65,34 +65,36 @@
...
         dictionary literal object, does not itself possess a docstring.
 
     Examples
     --------
     >>> from sunpy.sun import constants
-    >>> constants.constant('mass')
+    >>> constants.get('mass')
     <Constant name=u'Solar mass' value=1.9891e+30 uncertainty=5e+25 unit='kg' reference=u"Allen's Astrophysical Quantities 4th Ed.">
     """
+<<<<<<< HEAD
```
Comment (Cadair) [inline sunpy/sun/constants.py]: this is very bad!

**Comment by ehsteve** [comment]
@dpshelio @wafels @Cadair @ayshih ping

**Comment by wafels** [comment]
Travis is complaining about the 'disp' keyword, which is for displaying
results.  Don't think we need that at all.

**Comment by ehsteve** [comment]
ping @Cadair @wafels @ayshih @dpshelio passing now.

**Comment by dpshelio** [comment]
in a quick looks seems ok, but why is travis not happy?

**Comment by ehsteve** [comment]
@dpshelio It's a doc error which I totally don't understand. I am using iteritems from six and it is complaining about it in the API docs. @Cadair @ayshih?

File: sunpy/sun/constants.py [hunk 3]
```diff
@@ -62,222 +62,111 @@
 """
 
 from __future__ import absolute_import, division, print_function
+from astropy.table import Table
+from six import iteritems
 
 from sunpy.sun import _constants as _con # pylint: disable=E0611
```
Comment (Cadair) [inline sunpy/sun/constants.py]: The reason for the doc failiure is that this needs an `__all__` list.

**Comment by ehsteve** [comment]
Review please! @sunpy/sunpy-developers

**Comment by nabobalis** [comment]
I see no issue :+1:

File: sunpy/sun/constants.py [hunk 4]
```diff
@@ -62,222 +62,113 @@
 """
 
 from __future__ import absolute_import, division, print_function
+from astropy.table import Table
+from six import iteritems
```
Comment (bsipocz) [inline sunpy/sun/constants.py]: For all sorts of reasons six is bundled to sunpy and thus should be imported from ``sunpy.extern.six``.
e.g. see discussion in https://github.com/sunpy/sunpy/pull/1449

File: sunpy/sun/tests/test_constants.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
+from sunpy.sun import constants as con
+from astropy.constants import Constant
+import pytest
+from six import iteritems
```
Comment (bsipocz) [inline sunpy/sun/tests/test_constants.py]: ``sunpy.extern.six``

**Comment by Cadair** [comment]
@bsipocz good call, we need to add the `egg_info` build to travis, because presumeably that would have failed here?

**Comment by bsipocz** [comment]
@Cadair - Yes, I'm actually quite surprised that those builds are missing.

**Comment by Cadair** [comment]
lgtm, :+1:

**Pull request merged**